R/S4.R

```diff
@@ -51,6 +78,25 @@ S4_to_R7_class <- function(x, error_base = "") {
   }
 }
 
+R7_to_S4_class <- function(x) {
+  switch(class_type(x),
+    NULL = "NULL",
+    any = "ANY",
+    S4 = S4_class_name(x),
+    R7 = R7_class_name(x),
+    R7_base = double_to_numeric(x$class),
+    R7_S3 = x$class[[1]],
+    R7_union = "ANY",
+    stop("Unsupported")
```

Comment:
I guess you could have an R7 property of `class_missing`, and that would trigger this? That would be somewhat weird though. Everything else seems to be handled